src/main/java/carpet/script/CarpetEventServer.java

```diff
@@ -961,7 +961,7 @@ public CarpetEventServer(MinecraftServer server)
 
     public void tick()
     {
-        if (!TickSpeed.process_entities && TickSpeed.deepFreeze)
+        if (!TickSpeed.process_entities && TickSpeed.deeplyFrozen())
```

Comment:
well this is for 'schedule' events that rely on ticks - since tick events are paused in any frozen state, these should also be halted in regular frozen state.

Reply:
Wasn't it decided in the RFC at #581 that `schedule()` would use deep freeze instead of regular? (for things like "controlling" a freeze, for example?)

Reply:
well, 'schedule' is meant to run alongside the game - if entities are ticking, or blocks are interacting, schedule is something that follows that routine.

Reply:
Changed